assets/js/util/i18n.js

```diff
@@ -47,6 +47,28 @@ const durationFormat = ( seconds, options = {} ) => {
 		style: 'unit',
 	};
 
+	// Some browsers, e.g. Safari, throw a RangeError when options.style is
+	// not one of decimal, percent, or currency.
+	function testNumberFormat() {
+		try {
+			// test call to numberFormat
+			numberFormat( 10, {
+				...options,
+				unit: 'hour',
+			} );
+			return true;
+		} catch ( e ) {
+			return false;
+		}
+	}
+
+	const browserSupportsStyleUnit = testNumberFormat();
```

Comment:
Since we're declaring and using the function in one place, there's no need to create a function and then call it once.

Why not something like:

```
let browserSupportsStyleUnit = true;

try {
    // test call to numberFormat
    numberFormat( 10, {
        ...options,
        unit: 'hour',
    } );
} catch (error) {
    options.style = 'decimal';
    browserSupportsStyleUnit = false;
}
```